test-cases/manager/manager-regression-azure.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 360
+
+stress_cmd: "cassandra-stress write cl=QUORUM n=4000000 -schema 'replication(factor=3)' -mode cql3 native -rate threads=200 -pop seq=400000000..600000000"
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+post_behavior_db_nodes: "destroy"
+post_behavior_loader_nodes: "destroy"
+post_behavior_monitor_nodes: "destroy"
+use_preinstalled_scylla: true
+
+user_prefix: manager-regression
+space_node_threshold: 6442
+# ip_ssh_connections: 'public'
+
+azure_region_name: "eastus"
+backup_bucket_location: 'manager-backup-tests-us-east1'
```

Comment:
maybe location should directly refer to `eastus` instead `us-east1`?

Reply:
It already been used for a while, atm I don't want to mess with it

Reply:
but this is for azure test only, how will it affect other tests? Each cloud provider uses different value here for different regions.

Reply:
AFAIK this is only a name, and it was used this way to have a unified name, to all different backends, so it will be easy to set the tests, and move between different backends, for manager tests purpose

Reply:
in theory we should have a bucket per region, since now we are doing those backups across regions, and we pay more on the traffic.
we should also automate the process of creating/configuring those buckets, across all providers (and regions we use), I've started a bit of that in GCE as part of moving to the new project.